app/custom-hooks/custom-hooks.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
what i mean was 
cutom-hooks (folder)
inside the folder
1. useFetchNewsIDs.js (1st file)
2. useOnScrollBottom.js (2nd file)
so in total custom-hooks has 2 different files

Reply:
yeah it's a good idea, thanks